workflow_imaging/paths.py

```diff
@@ -9,10 +9,8 @@ def get_imaging_root_data_dir():
 
 def get_scan_image_files(scan_key):
     # Folder structure: root / subject / session / .tif (raw)
-    data_dir = get_imaging_root_data_dir()
-    subj_dir = data_dir / scan_key['subject']
-    sess_datetime_string = scan_key['session_datetime'].strftime('%Y%m%d_%H%M%S')
-    sess_dir = subj_dir / sess_datetime_string
+    from workflow_imaging.pipeline import Session
+    sess_dir = pathlib.Path((Session.Directory & scan_key).fetch('session_dir')[0])
```

Comment:
you can do:
```
sess_dir = pathlib.Path((Session.Directory & scan_key).fetch1('session_dir'))
```

Reply:
also, let's do a check if directory path exist and raise error appropriately

Reply:
thanks, updated